packages/v4/patternfly-docs/content/design-guidelines/components/banner/banner.md

```diff
@@ -7,33 +7,33 @@ section: components
- 
-|Type     |Background color    |Text color  | Usage
-|---------|---------------|----------------| ------------------ |
-|Default  |Grey <br />(--pf-global--BackgroundColor--dark-400) | Black |Use for generic messages with no associated severity
-|Info     |Blue <br />(--pf-global--info-color--100)|Black| Use for general information messages |
-|Danger  |Red <br />(--pf-global--danger-color--100) |White| Use for high severity messages
-|Success|Green <br />(--pf-global--success-color--100) | White | Use for positive, success messages|
-|Warning |Orange <br />(--pf-global--warning-color--100) | Black | Use for warning or medium severity messages |
-<br />
-PatternFly suggests that users adopt one of these 5 colors, as they’ve been tested against their text colors for accessibility. However, if colors outside of these suggestions are used, we encourage using discretion when selecting the inner text color.
+<img src="./img/banner-types.png" alt="5 different types of banners"/>
+ 
+|Type |Background color |Text color | Usage |
+|-|-|-|-|
+| Default | `--pf-global--BackgroundColor--dark-400` | White | Generic messages with no associated severity |
+| Blue | `--pf-global--info-color--100` | Black | General information messages |
+| Red | `--pf-global--danger-color--100` | White | High severity messages
+| Green | `--pf-global--success-color--100` | White | Positive, success messages |
+| Gold | `--pf-global--warning-color--100` | Black | Warning or medium severity messages |
+
+PatternFly suggests that users adopt one of these 5 colors, as they’ve been tested with their text colors for accessibility. However, if colors outside of these suggestions are used, we encourage using discretion when selecting the text color.
 
 ### When to use
  
 Banners are flexible and can be used for any content and for several use cases outlined in the following examples.
  
 - **To indicate that an application/site is offline.** In this example, a banner is used to indicate to the user that the site is undergoing maintenance and that they will have limited functionality as a result. The banner is shown at the top and bottom to ensure that the user does not miss this disclaimer. Since this is a general, non-severe message, a default banner was used.
  
-   <img src="./img/example-1.png" alt="Example of default banner in use" width="990"/>
+   <img src="./img/example-1.png" alt="Example of default banner in use" />
  
-- **To indicate that an application upgrade is available.** In this example, the user is notified of an available upgrade through a banner at the top of the page. Since the use case here is to convey information, an informational banner was used.
+- **To indicate that an application upgrade is available.** In this example, the user is notified of an available upgrade through a banner at the top of the page. Since the use case here is to convey information, an blue status banner was used.
```

Comment:
```suggestion
- **To indicate that an application upgrade is available.** In this example, the user is notified of an available upgrade through a banner at the top of the page. Since the use case here is to convey information, a blue status banner was used.
```